XMonad/Layout/BorderResize.hs

```diff
@@ -58,27 +58,24 @@ data BorderInfo = BI { bWin :: Window,
 
 type RectWithBorders = (Rectangle, [BorderInfo])
 
-newtype BorderResize a = BR (M.Map Window RectWithBorders) deriving (Show, Read)
+data BorderResize a = BR Dimension (M.Map Window RectWithBorders) deriving (Show, Read)
 
-brBorderSize :: Dimension
-brBorderSize = 2
-
-borderResize :: l a -> ModifiedLayout BorderResize l a
-borderResize = ModifiedLayout (BR M.empty)
+borderResize :: Dimension -> l a -> ModifiedLayout BorderResize l a
```

Comment:
Either this should be listed as a breaking change on `CHANGES.md`, or the variant that takes a parameter should be a separate function. In the latter case, the Haskell convention is that the variant's name ends with a tick (that is, `borderResize'`). The normal one would then be implemented as the ticked one with `brBorderSize` as the parameter.

Reply:
Maybe `borderResizeNear` or something like that would be a better name as well (not a fan of nondescript primes :)

Reply:
I pushed a new commit to my fork: [9ff92606d83e9421fd60a36e58ffd3eaeef35338](https://github.com/Quoteme/xmonad-contrib/commit/9ff92606d83e9421fd60a36e58ffd3eaeef35338)
This adds a variant as you suggested.